image_services/egis/publishing_MDCS_to_EGIS.md

```diff
@@ -0,0 +1,111 @@
+> 
+> While other paths in the list may be broken, the only one that requires a fix in this workflow is the one that impacts the pixel cache. It is not, however, problematic to set all of the broken file paths to local paths if you are not sure which one requires the change.
+>
+>Note that the repaired paths may not include the PixelCache designation in the path following the repair, but rather truncate the listed path to a parent directory beyond which the paths match. 
+> 
+>For example, the repairs illustrated in step 3 above did not retain the full path after the repairs were complete; the path listed in the mosaic dataset includes only the portion of the path that didn't originally match:
+>
+>- Old path: `G:\Projects\2209_ImageServices\ImageServices\PixelCache`
+>- New path: `C:\Users\hjkristenson\Documents\ImageServer\ImageServices\PixelCache`
+>- Path currently displayed: `C:\Users\hjkristenson\Documents\ImageServer\ `
+>
+>If using GSSICB files from the original source computer rather than those archived in s3://asf-gis-services/GSSICB/, there are two scenarios: 
+>1. COH12_VV services use relative paths, so the directory to repair is `C:\Users\hjkristenson\PycharmProjects\hyp3-nasa-disasters`
+>   * You can use any path on your local computer to replace the broken path, but be aware that a folder structure will be generated to next within it to mirror the original directory structure, including subdirectories in the hyp3-nasa-disasters directory, ending with a PixelCache directory
+>2. All of the other GSSICB services use the following path for the pixel cache: `G:\Projects\2209_ImageServices\ImageServices\PixelCache`
+>   * In this case, it's easiest to set a local path that ends in a PixelCache directory
+
+
+## Edit the Metadata for the Mosaic Dataset
+
+1. Right-click the mosaic dataset in the Catalog pane, and select Edit Metadata
+![Edit Metadata Figure](images/EditMetadata.PNG)
+2. Populate the Title, Tags, Summary, Description and Credits sections and click the Save button at the top of the window
+   * Refer to the sample metadata files for the [COH12](sample_metadata_COH12.txt) and [AMP](sample_metadata_AMP.txt) datasets for content
+
+## Publish the Mosaic Dataset as an Image Service
+
+1. Right-click the mosaic dataset in the Catalog pane and select Share As Web Layer
+![Share As Web Layer](images/ShareAsWebLayer.PNG)
+2. Set the options in the General tab
+![Share As Web Layer Dialog](images/ShareAsWebLayerDialog.PNG)
+   1. Set the Portal Folder to GSSICB
+   2. Set the Server to https://url_to_gis_server/image (this is not the default in the Server dropdown menu, and if this is not changed, there will be a licensing error)
+   3. Set the server Folder to GSSICB
+   4. Select Everyone and Earthdata GIS under the Share options
```

Comment:
```suggestion
   4. Select Earthdata GIS under the Share options
```
We can no longer share with everyone, right?

Reply:
Good point. I'll make an edit.